static/js/script.js

```diff
@@ -0,0 +1,3 @@
+	jQuery(document).ready(function($){
+			$(".datetime-widget").datetimepicker({ format:'Y-m-d H:i'	});
+		});
```

Comment:
Missing newline, and you don't need tab before jQuery
There's whitespace after `format:`